CHANGES.md

```diff
@@ -48,21 +69,19 @@ To be released.
 
  -  The following store URI schemes are deprecated:  [[#1573], [#1955]]
 
-     - `default`: Use `default+file` instead.
-     - `rocksdb`: Use `rocksdb+file` instead.
-
```

Comment:
Why were these lines removed?

Reply:
I'm fixing it currently. I accidently removed some links also while rebasing. 😅